static/app/components/search/sources/commandSource.tsx

```diff
@@ -65,6 +66,27 @@ const ACTIONS: Action[] = [
   },
 ];
 
+const hostname = new URL(
+  usingCustomerDomain && window?.__initialData?.customerDomain?.organizationUrl
+    ? window.__initialData.customerDomain.organizationUrl
+    : window.__initialData?.links?.sentryUrl
+).host;
+
+window?.__initialData?.isOnPremise === false &&
+  process?.env?.NODE_ENV &&
+  process.env.NODE_ENV !== 'production' &&
+  ACTIONS.push({
+    title: t('Open in Production'),
+    description: t('Open the current page in sentry.io'),
+    requiresSuperuser: false,
+    action: () => {
+      const url = new URL(window.location.toString());
```

Comment:
@narsaynorath is `url.protocol` guaranteed to be `https`?

Reply:
Good point, I also pulled out the protocol from the customer domain. Considering this is for our internal usage, I think our dev envs and prod always use `https` but it's easy to just be safe and add it.